tests/testdtls.nim

```diff
@@ -0,0 +1,45 @@
+# Nim-WebRTC
+# Copyright (c) 2024 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+{.used.}
+
+import chronos
+import ../webrtc/udp_transport
+import ../webrtc/stun/stun_transport
+import ../webrtc/dtls/dtls_transport
+import ../webrtc/dtls/dtls_connection
+import ./asyncunit
+
+suite "DTLS":
+  teardown:
+    checkLeaks()
+
+  asyncTest "Simple Test":
```

Comment:
Could you please use a more descriptive name for what the test is accomplishing? If it is hard to do, it could be a symptom it is testing too many things and should be split.

Reply:
Done